modules/digiTrustIdSystem.js

```diff
@@ -102,6 +102,13 @@ function initDigitrustFacade(config) {
       callCount: 0,
       initCallback: null
     },
+    initialize: function (config) {
+      /* This method is present to catch an edge case where DigiTrust publishers transition
+       * to Prebid with minimal integration, but have not removed old DigiTrust.initialize call.
+       * It should not be used */
+      utils.logInfo('Warning - call to Digitrust.initialize within a Prebid integration. Please initialize through pbjs.configure with the userSync option');
+      initializeDigiTrust(config);
```

Comment:
What does this call to `initializeDigiTrust` do? In order to reach this part of the code, `window.DigiTrust` has already been set to the Prebid facade, correct?

Reply:
This passes the config object along to set member ID, if it hasn't been set properly. The code is designed to work if the underlying DigiTrust library is present (legacy migration or blended deployment) or if it is not present (prebid integration only). In the majority of cases this call does nothing. In some rare cases - mostly from migrations that are in-process - this call passes the member ID into the system. We saw this in the wild generating errors and wanted to fix it.

Reply:
@goosemanjack Sorry, but I'm still not understanding this. The `initialize` function you've defined is within `fascade`. So, it looks like it will only be called if `window.DigiTrust` has been set to the `fascade`. In this case, `initializeDigiTrust` should always do nothing since `dt.isClient` will be `true`.
https://github.com/prebid/Prebid.js/blob/da652e095ffe91dc7df3d6c11c1f51999afe4c51/modules/digiTrustIdSystem.js#L311